codebases/signalen.html

```diff
@@ -0,0 +1,136 @@
+---
+layout: codebase
+title: Codebases we work with - Signalen
+---
+
+<div class="cb-section">
+  <div>
+    <h1 class="title">Codebases we work with</h1>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-maturity-detail-header">
+    <div class="cb-maturity-detail-title cb-maturity-s1">
+      <div class="cb-logo"><img src="../photos/codebases/signalen/signalen_logo_small.png" alt="Signalen logo"></div>
```

Comment:
In the [guidelines for alt-text](https://www.digg.se/om-oss/nyheter/2021/textalternativ-for-bilder--information-nar-inte-bilden-kan-visas) from the Swedish Agency for digital government they have an explicit example of not using the alt-text "Agency logo" but rather recommend just "Agency" as the text. Not sure if this is a local guideline or universal.

Reply:
[This website](https://webaim.org/techniques/alttext/#logos) seems to recommend the same.